Oxford-EEBO-TCP.js

```diff
@@ -0,0 +1,232 @@
+{
+	"translatorID": "62151d94-6fdf-442f-be29-94bbd24fcd09",
+	"label": "Oxford-EEBO-TCP",
+	"creator": "pierwill",
+	"target": "[^]*[ota|tei.it|purl].ox.ac.uk/[tcp|id]/*",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-02-15 05:20:51"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+    return "book";
+}
+
+function doWeb(doc, url) {
+    
+    newItem = new Zotero.Item("book");
+    
+    if (url.includes("http://tei.it.ox.ac.uk/tcp/Texts-HTML/free/") == true) {
+	html(doc, url);
+    } else {
+	catalogue(doc, url);
+    }
```

Comment:
The following scraping from that page looks very fragile. Moreover, there should be the corresponding TEI file available, e.g. http://tei.it.ox.ac.uk/tcp/Texts-TEI/free/A20/A20631.xml which has the information in structured form and we should therefore use instead.

Reply:
Stopped here with the review therefore.